### ISSUE TITLE
Tool ignores Bookmark save file (Temporary)

### ISSUE THREAD

**Issue by Jacien** [SciresM/FEA-Save-Tool#4]
Hello,

I can understand if you never see this cause of the busy work going on with the Switch, but I recently was replaying Fire Emblem Awakening and saw this tool and wanted to mess around with a fresh save file.  Wanted to try editing a Chapter 1 Bookmark, however it turns out the tool does not decompress that.

When on Classic mode, instead of saving a mid-chapter file (Map0, Map1), it creates a temporary file named "Temporary".  The tool decompresses all other files without issue (Chapter0, Chapter1, Chapter2, Global, Map0, Map1).

I believe the fix is inside Form1.cs, but I don't know for sure.

Here's an example in Dropbox, showing the dumped folder that has Chapter0, Global, and Temporary, but in the decompressed folder it only has Chapter0 and Global. https://www.dropbox.com/sh/pxqxl7mmigyzh55/AABUNUCOJqgJDvgn4LuOhUKEa?dl=0

I'll try poking through things and asking around if anybody can help out if you're busy or something.  I understand this stuff is super old, and I don't want to be blamed for any kind of Switch delays, haha.

Take care.

**Comment by Jacien** [comment]
Okay, so I did some clever thinking (me so smart).

I renamed the Temporary file to Chapter1, and it decompressed perfectly. (I also tested it by renaming it to Map1, and it failed).

I confirmed it by opening up in a hex editor and searching for Chrom's hair color.

**Comment by Jacien** [comment]
Did not mean to push that, lol.

![image](https://user-images.githubusercontent.com/38120819/38402265-f7179be6-3928-11e8-80ff-5f9259f6e613.png)

So after I decompressed it by renaming it to Chapter1, I searched for (505C81FF) which is Chrom's hair color in the character block, and it popped up without issue.  So with that logic I can recompress it, then renamed it back to Temporary and it'll work fine.

So in short, the decompression that's used for Chapter# works perfectly for the Temporary file.  So I just have to rename it to Chapter0, Chapter1, or Chapter2.

So there's a workaround, so it's not as critical, but it's good to know.  Thank you for all the work you do, and I wish you took donations.  Cause I can't be spending thousands on NSFW commissions all the time, I gotta mix it up.
